web/src/components/molecules/View/viewMenuItem.tsx

```diff
@@ -71,31 +71,19 @@ const ViewsMenuItem: React.FC<Props> = ({
   ];
 
   return (
-    <StyledDropdownButton
-      trigger={["click"]}
-      type="text"
-      icon={<Icon icon="more" />}
-      menu={{ items: children }}>
-      {t(view.name)}
-    </StyledDropdownButton>
+    <Wrapper>
+      {view.name}
+      <Dropdown trigger={["click"]} menu={{ items: children }}>
+        <Icon icon="more" size={16} style={{ marginRight: 0 }} />
```

Comment:
Can you use styled components instead of inline styles.
Inline styles are slow and they decrease the performance. Try to avoid it as much as possible.
```suggestion
        <Icon icon="more" size={16} />
```

Reply:
Okay!